browserpass.rb

```diff
@@ -1,8 +1,8 @@
 class Browserpass < Formula
   desc "Host application for browser extension providing access to your password store"
   homepage "https://github.com/browserpass/browserpass-native"
-  url "https://github.com/browserpass/browserpass-native/releases/download/#{version}/browserpass-darwin64-#{version}.tar.gz"
   version "3.0.7"
+  url "https://github.com/browserpass/browserpass-native/releases/download/#{version}/browserpass-darwin64-#{version}.tar.gz"
```

Comment:
So based on current brew style/audit failure, we don't even need this explicit `version` attribute anymore. I guess let's clear it up - @joshtch would you mind removing the `version` attribute entirely, and updating the `url` to use `3.0.7` explicitly?

Reply:
Alright, got it. (See new version.) I also ran `brew audit` again to check for other issues, and it looks okay now.